cmd/fyne/internal/commands/release.go

```diff
@@ -12,8 +12,8 @@ import (
 	"fyne.io/fyne/v2"
 	"fyne.io/fyne/v2/cmd/fyne/internal/mobile"
 	"fyne.io/fyne/v2/cmd/fyne/internal/templates"
+	cli "github.com/urfave/cli/v2"
```

Comment:
This change does not seen correct. The name of the package is already `cli`.

Reply:
I think it is correct since all other imports you have in code use `v2`, see
- https://github.com/fyne-io/fyne/blob/develop/cmd/fyne/main.go#L10
- https://github.com/fyne-io/fyne/blob/develop/cmd/fyne/internal/commands/install.go#L14
- https://github.com/fyne-io/fyne/blob/develop/cmd/fyne/internal/commands/build.go#L13
etc.

I can remove `cli` alias though and leave only `"github.com/urfave/cli/v2"`

Reply:
I only meant that the alias was incorrect. Sorry for any confusion

Reply:
Moreover, I see the following build error if I do not use `v2`:
```
go build
package fyne.io/fyne/v2/cmd/fyne
	imports fyne.io/fyne/v2/cmd/fyne/internal/commands
	imports github.com/urfave/cli: no Go files in /Users/vk/tmp/fyne/vendor/github.com/urfave/cli
```
So, based on your go.mod file it is required to have v2, see https://github.com/fyne-io/fyne/blob/develop/go.mod#L28

Therefore, I only removed alias `cli` and left `"github.com/urfave/cli/v2"`